ui/src/Panels/Producer/Producer.scss

```diff
@@ -0,0 +1,21 @@
+.Producer {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    min-height: 100vh;
+    background-color: $ui-01;
+
+    &__count {
+        margin-top: 10%;
+    }
+    &__control {
+        display: flex;
+        justify-content: space-between;
+    }
+    &__Message {
```

Comment:
uber picky, but could this be lower case?

Reply:
Done.